Directory.Build.props

```diff
@@ -0,0 +1,15 @@
+<Project>
+
+  <PropertyGroup>
+    <TargetFrameworkNETStandard>netstandard2.0</TargetFrameworkNETStandard>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <BuildOutputDirectory>$(MSBuildThisFileDirectory)bin\$(Configuration)\</BuildOutputDirectory>
+    <BootstrapOutputDirectory>$(MSBuildThisFileDirectory)bin\Build$(Configuration)\</BootstrapOutputDirectory>
+    <TestOutputDirectory>$(MSBuildThisFileDirectory)bin\Test$(Configuration)\</TestOutputDirectory>
+    <BootstrapTasksAssembly>$(BootstrapOutputDirectory)$(TargetFrameworkNETStandard)\Xamarin.Android.Tools.BootstrapTasks.dll</BootstrapTasksAssembly>
+    <PrepTasksAssembly>$(BootstrapOutputDirectory)$(TargetFrameworkNETStandard)\xa-prep-tasks.dll</PrepTasksAssembly>
+    <!-- Copy PackageReference content to OutputDir for our build tasks, tests, and installer creation logic. This no longer happens by default in short-form projects. -->
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+  </PropertyGroup>
```

Comment:
Maybe TODO on another PR. Can we kill `Configuration.props` in favor of the new hotness: `Directory.Build.props`?

Reply:
Yeah I definitely think that's a good longer term goal. We may want to do it in a couple of stages though, I imagine the diff is going to be pretty ugly (not to mention the changes required in external sources...).